### ISSUE TITLE
Мультиязычность

### ISSUE THREAD

**Issue by foza** [sr-projects/speedrunner-cms#3]
Реализовано мультиязычность?

**Comment by alinskydev** [comment]
Да, DBTranslation для констант и есть behavior для ActiveRecord. Пример использования можно посмотреть в модели Page.
